platform-core-module/src/main/java/com/softicar/platform/core/module/maintenance/session/SessionManager.java

```diff
@@ -0,0 +1,49 @@
+package com.softicar.platform.core.module.maintenance.session;
+
+import com.softicar.platform.ajax.document.AjaxDocument;
+import com.softicar.platform.ajax.framework.listener.AjaxSessionListener;
+import com.softicar.platform.core.module.CoreRoles;
+import com.softicar.platform.core.module.ajax.session.SofticarAjaxSession;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpSession;
+
+public class SessionManager {
```

Comment:
this class code looks very procedural, suggestion: make the methods non-static and add field `ServletContext context` initialized in the constructor?